client_internals_it_test.go

```diff
@@ -421,6 +423,32 @@ func clientInternalEncodeDataTest(t *testing.T) {
 	})
 }
 
+func clientClusterConnectionConfigRetryTimeTest(t *testing.T) {
+	// TODO: Adapt this test for t.Parallel()
+	//t.Parallel()
+	ctx := context.Background()
+	const ASSERTION_SECONDS = 30
```

Comment:
Could you change this line to `const AssertionSeconds = 30` to make it in line with Go conventions?

Reply:
Fixed at https://github.com/hazelcast/hazelcast-go-client/pull/861/commits/830865005e9c8f61b0599f85a7fb25de30d862fc